examples/05-plotting/02-solution_combination.py

```diff
@@ -0,0 +1,90 @@
+# Next, open an example and print out the ``model`` object.  The
+# ``Model`` class helps to organize access methods for the result by
+# keeping track of the operators and data sources used by the result
+# file.
+#
+# Printing the model displays:
+#
+# - Analysis type
+# - Available results
+# - Size of the mesh
+# - Number of results
+#
+model = dpf.Model(examples.msup_transient)
+print(model)
+###############################################################################
+# Get the stress tensor and connect time scoping.
+# Make sure to define ``Nodal`` as requested location,
+# as the labels are supported only for Nodal results.
+#
+stress_tensor = model.results.stress()
+time_scope = dpf.Scoping()
+time_scope.ids = [1, 2]
+stress_tensor.inputs.time_scoping.connect(time_scope)
+stress_tensor.inputs.requested_location.connect("Nodal")
+
+###############################################################################
+# This code performs solution combination on two load cases.
+# =>LC1 - LC2
+# You can access individual loadcases as the fields of a fields_container for `stress_tensor`
+#
+# LC1: stress_tensor.outputs.fields_container.get_data()[0]
+# LC2: stress_tensor.outputs.fields_container.get_data()[1]
+#
+# Scale LC2 to -1
+field_lc2 = stress_tensor.outputs.fields_container.get_data()[1]
+stress_tensor_lc2_sc = dpf.operators.math.scale(field=field_lc2,
+                                                ponderation=-1.0)
+###############################################################################
+# Add load cases
+#
+field_lc1 = stress_tensor.outputs.fields_container.get_data()[0]
+stress_tensor_combi = dpf.operators.math.add(fieldA=field_lc1,
+                                             fieldB=stress_tensor_lc2_sc)
+###############################################################################
+# Principal Stresses are the Eigenvalues of the stress tensor.
+# Use ``principal_invariants`` to get S1, S2 and S3
+#
+p_inv = dpf.operators.invariant.principal_invariants()
+p_inv.inputs.field.connect(stress_tensor_combi)
+###############################################################################
+# Print S1 - Maximum Principal stress
+#
+print(p_inv.outputs.field_eig_1().data)
+###############################################################################
+# Get the meshed region
+#
+mesh_set = model.metadata.meshed_region
+###############################################################################
+# Plot the results on the mesh.
+# ``label_text_size`` and ``label_point_size`` control font size of the label.
+#
+plot = DpfPlotter()
+plot.add_field(p_inv.outputs.field_eig_1(),
+               meshed_region=mesh_set,
+               show_min=False,
+               show_max=True,
+               label_text_size=40,
+               label_point_size=15,
+               )
+###############################################################################
+# Show figure
+#
+plot.show_figure(show_axes=True)
```

Comment:
@anslpa this line doesn't work in the documentation [build ](https://dev.azure.com/pyansys/pyansys/_build/results?buildId=3153&view=logs&j=f356c297-46ae-5fb8-4401-8a57dd17255c&t=1060c213-875e-5ba9-3b31-ff5609a3bc3c), does it ring a bell? I am thinking of something to do with the plotting off screen maybe? This example runs fine in the WindowsJob/Run examples

Reply:
It's because of the split between cells. The image scraper for `pyvista` and `sphinx_gallery` runs at the end of each cell and it automatically clears out the plotter.